### PR TITLE
OMERO 5 6 0

### PR DESCRIPTION
**Pull request by joshmoore**
Mostly waiting on the final text from the announcement.

### CODE REVIEW

**Comment by joshmoore** [comment]
Python download blurb for review: 
Screen Shot 2020-01-14 at 16 47 14

File: _posts/2020-01-15-omero-5-6-0.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title: Release of OMERO 5.6.0 with Python 3
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 6.3.1
```
Comment (mtbc) [inline _posts/2020-01-15-omero-5-6-0.md]:
```suggestion
intro-blurb: The OME team is pleased to announce the release of OMERO 5.6.0.
```

File: omero/downloads/index.html [hunk 2]
```diff
@@ -116,9 +129,12 @@ <h3 class="text-center">OMERO Python</h3>
                     <div class="card-section">
                         <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_python.svg">
                         <h5>Python (Ice 3.6)</h5>
-                        <h6>OMERO.py-{{ site.omero.version }}-ice36-{{ site.omero.build }}.zip</h6>
-                        <p class="card-caption">OMERO.py can be used to install OMERO.web decoupled from the OMERO.server, see the <a href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/unix/install-web/web-deployment.html">OMERO.web documentation</a> for details.</p>
-                        <a href="https://downloads.openmicroscopy.org/omero/{{ site.omero.version }}/artifacts/OMERO.py-{{ site.omero.version }}-ice36-{{ site.omero.build }}.zip" class="button hollow small"><i class="fa fa-download"></i> Download (8.7 MB)</a>
+                        <p class="card-caption">
+                        OMERO.py libraries can be downloaded from PyPI
+                        or from the ome conda channel.
```
Comment (mtbc) [inline omero/downloads/index.html]:
```suggestion
                        or from the <a href="https://anaconda.org/ome">ome conda channel</a>.
```

**Comment by joshmoore** [comment]
Screen Shot 2020-01-15 at 11 27 46
Additionally dropped the source code section

**Pull request merged**